test/server/utils/new-line-replacer.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function replaceNewLines (content) {
```

Comment:
As I stated previously let's not make utils for this single method, Just keep it inside tests. It's naming is still misleading, because it's very specific to the tests and form data.